api/scripts/wazuh-apid.py

```diff
@@ -15,23 +15,29 @@
 API_MAIN_PROCESS = 'wazuh-apid'
 API_LOCAL_REQUEST_PROCESS = 'wazuh-apid_exec'
 API_AUTHENTICATION_PROCESS = 'wazuh-apid_auth'
+API_SECURITY_EVENTS_PROCESS = 'wazuh-apid_events'
 
 
 def spawn_process_pool():
-    """Import necessary basic Wazuh SDK modules for the local request pool and spawn child."""
-    from wazuh import agent, manager  # noqa
-    from wazuh.core import common  # noqa
-    from wazuh.core.cluster import dapi  # noqa
+    """Spawn general process pool child."""
 
     exec_pid = os.getpid()
     pyDaemonModule.create_pid(API_LOCAL_REQUEST_PROCESS, exec_pid)
 
     signal.signal(signal.SIGINT, signal.SIG_IGN)
 
 
+def spawn_events_pool():
+    """Spawn events process pool child."""
+
+    events_pid = os.getpid()
+    pyDaemonModule.create_pid(API_SECURITY_EVENTS_PROCESS, events_pid)
```

Comment:
Beware that adding a new process can affect many different places, it is very important to test this in deployments, etc!

Reply:
How do you suggest handling this? Do we need to open a new issue to track it?

Reply:
Yes @nico-stefani, you can open a new issue inside the epic if you prefer. As long as this is tested before merging the epic branch, is fine.